CHANGELOG.md

```diff
@@ -107,6 +107,8 @@
 
 ## v0.141.6
 
+_29 Sep 2025_
```

Comment:
I also updated the changelog to include the dates for the past releases

<img width="914" height="788" alt="Screenshot 2025-10-04 at 09 11 04" src="https://github.com/user-attachments/assets/fb71a142-7f91-4bf7-96c3-a1727dbec625" />